src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -1280,12 +1280,17 @@ internal static BoundExpression NullOrDefault(TypeSymbol typeSymbol, SyntaxNode
                 case RefKind.Out:
                     refKind = RefKind.Ref;
                     break;
+
+                // "in" arguments are spilled by-value, while "strict-in" arguments are
+                // spilled by-ref. It's the responsibility of the caller to determine
+                // whether or not their argument can be spilled by-ref.
```

Comment:
It's unclear if the second sentence refers to explicit or implicit in arguments.